### PR TITLE
Add page exclusive for pt-br content

### PR DESCRIPTION
**Pull request by afucher**
Propose a pattern to create pages with curated content in other languages than english.

- Create folder structure to handle languages: `/content/<language>`
- Add a index to all available language content (I opted to add english as an option and link to the main content, to be explicit)
- Fix the link in the main page to the repository, it was pointing to a totally different repository (probably the one that was inspiration for this one).

### CODE REVIEW

**Comment by yuhattor** [comment]
Here's my suggestion
@spier @pmalirz 

- moving file from `/content/pt-br/README.md` to `README.pt-br.md`. People will be easily find the contents.
- Inviting @afucher and @fer-correa and Eneri to the repository and adding them as codeowners for the README.pt-br.md file

What do you think? If you can agree on that, I will ad them as codeowners

**Comment by spier** [comment]
I agree, let's start with the most simple possible structure, and see how that works.
i.e. I am in favor of the solution that @yuhattor proposed 👍

**Comment by afucher** [comment]
Done! @spier @yuhattor can you check please?

File: README.md [hunk 1]
```diff
@@ -11,7 +11,11 @@ A curated list of awesome **[InnerSource](https://innersourcecommons.org/)** res
 [InnerSource](https://innersourcecommons.org/) is a software development strategy that applies open source best practices to proprietary code. InnerSource can help establish an open source culture within an organization while retaining software for internal use. Teams use InnerSource to increase visibility, strengthen collaboration and break down silos. Through the set of practices, InnerSource helps companies to simplify, standardize and make cost savings globally and across departments.
 
 <!--lint ignore double-link-->
-This [awesome](https://github.com/sindresorhus/awesome) repository is maintained by the [InnerSource Commons Community](https://innersourcecommons.org/).
+This [awesome](https://github.com/InnerSourceCommons/awesome-innersource) repository is maintained by the [InnerSource Commons Community](https://innersourcecommons.org/).
+
+### Contents available in:
```
Comment (spier) [inline README.md]:
```suggestion
### Contents available in:

```

File: README.pt-br.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+<!--lint ignore double-link-->
+[<img src="../../assets/images/innersource-logo.png" align="right" width="100" alt="InnerSource Commons">](https://innersourcecommons.org/)
+
+<!--lint ignore double-link-->
+[InnerSource](https://innersourcecommons.org/) é uma estratégia de desenvolvimento de software que aplica melhores práticas de open source em código proprietário. InnerSource pode ajudar a estabelecer uma cultura open source na organização enquanto retém o código para uso interno. Times usam InnerSource para aumentar a visibilidade, fortalecer colaboração e quebrar silos. Através de um conjunto de práticas, InnerSource ajuda empresas a simplificar, padronizar e reduzir custos globalmente e entre departamentos.
+
+<!--lint ignore double-link-->
+Esse [incrível](https://github.com/InnerSourceCommons/awesome-innersource) (_awsome_) repositório é nabtudi pela [Comunidade InnerSource Commons](https://innersourcecommons.org/).
+
```
Comment (spier) [inline README.pt-br.md:13]:
```suggestion
```

**Review by spier — APPROVED** [review]
LGTM

left minor formatting suggestions inline

**Comment by afucher** [comment]
@spier commited your suggestions 😇  can you review again please?

File: README.pt-br.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+Uma lista de recursos incríveis sobre **[InnerSource](https://innersourcecommons.org/)** (livros, artigos, blogs, projetos e mais).  
+
+> If you do not speak portuguese, check [other languages availables](./README.md#contents)
```
Comment (spier) [inline README.pt-br.md]:
```suggestion
> If you do not speak Portuguese, check [other languages available](./README.md#contents)
```

Found another small typo. Sorry :)

**Review by spier — COMMENTED** [review]
Found a tiny something :)

File: README.pt-br.md (hunk 3 above)
Comment (afucher) [reply README.pt-br.md]: Do not need to say sorry, this is the reason to ask for review 😝

**Comment by afucher** [comment]
Done @spier !

**Review by spier — APPROVED** [review]
Looks great!

Should I merge? I am pretty sure @pmalirz won't object :)

**Comment by afucher** [comment]
I'll fix the lint issues and update the PR. I also created an issue to discuss some improvements about the lint in the project: https://github.com/InnerSourceCommons/awesome-innersource/issues/15

**Comment by spier** [comment]
Thank you for your contribution @afucher! ❤️

**Review by pmalirz — APPROVED** [review]

**Pull request merged**